docs/user/alerting/alerting-setup.asciidoc

```diff
@@ -113,16 +113,13 @@ For more information on configuring roles that provide access to features, go to
 Rules are authorized using an API key.
 Its credentials are used to run all background tasks associated with the rule, including condition checks like {es} queries and triggered actions.
 
-If you create or edit a rule in {kib}, an API key is created that captures a snapshot of your privileges at the time of the edit.
-The following actions regenerate the API key in {kib}:
-
-* Creating a rule
-* Updating a rule
+When you create or import a rule in {kib}, an API key is created that captures a snapshot of your privileges.
+Likewise when you update a rule, it updates the API key with a snapshot of your privileges at the time of the edit. 
 
 When you disable a rule, it retains the associated API key which is reused when the rule is enabled.
-If the API key is missing when you enable the rule (for example, in the case of imported rules), it generates a new key that has your security privileges.
+If the API key is missing when you enable the rule, it generates a new key that has your current security privileges.
```

Comment:
> If the API key is missing when you enable the rule...

@mikecote Is this sentence still accurate now that I've moved the "import" scenario to an earlier paragraph?

Reply:
Yes this is accurate. The import scenario will create the rule as disabled then the user will need to enable it like this scenario.

Reply:
> import a rule in {kib}

I think when we import a rule into Kibana, there is no API key (set to `null` on export), and we actually post a message that the rules need to be enabled to start running.  Enabling them will create the API key, using the privs of the user enabling the rule.

Reply:
Thanks for the clarifications!